### DISCUSSION TITLE
How to remove "Constraints physics"?

### DISCUSSION THREAD

**Discussion by vitalspace** [enable3d/enable3d#306] [Help / Questions]
How to remove "Constraints physics"?

i want to separate 2 bodies after some time;

i try with ```this.physics.destroy``` but no work.

ty...

**Comment by yandeu** [comment]
I found this https://pybullet.org/Bullet/phpBB3/viewtopic.php?p=34822&sid=5a3c9751ee284bf1da0ff5de39a93141#p34822

**Reply by vitalspace** [reply]
I'm going to try it

**Reply by yandeu** [reply]
Did it work?

**Reply by vitalspace** [reply]
i found another way to not use "Constraints". anyway ty.
